tests/modules/programs/mpv/mpv-invalid-settings.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I'm not sure whether the test runner supports tests where the configuration fails to build (intentional failure).

@rycee ?

Reply:
I took this example from here: https://github.com/nix-community/home-manager/blob/master/tests/modules/programs/rofi/assert-on-both-theme-and-colors.nix#L20-L30

It seems that `home.file.result.text` line makes it to output the result of a failed assertion.